src/main/java/gov/nasa/pds/api/engineering/elasticsearch/ElasticSearchUtil.java

```diff
@@ -24,6 +25,13 @@ static public String jsonPropertyToElasticProperty(String jsonProperty) {
 		
 	}
 	
+	static private void addReference (ArrayList<Reference> to, String ID, String baseURL)
```

Comment:
Can you move the method after the next one. I would like to keep next to each other jsonPropertyToElasticProperty and elasticPropertyToJsonProperty.

Reply:
Done